R/PipeOpTaskSurvClassif.R

```diff
@@ -0,0 +1,151 @@
+#' @title PipeOpTaskSurvClassif
+#' @name mlr_pipeops_trafotask_survclassif
+#' @template param_pipelines
+#'
+#' @description
+#' Transform [TaskSurv] to [TaskClassif][mlr3::TaskClassif] by creating multiple
```

Comment:
Can we please write a more general description here and not include the parameter names (only in a later section as we have it)?